packages/realm/src/TypeHelpers.ts

```diff
@@ -73,7 +72,7 @@ export function toArrayBuffer(value: unknown, stringToBase64 = true) {
 
 /** @internal */
 export type TypeHelpers<T = unknown> = {
-  toBinding(value: T, createObj?: ObjCreator): binding.MixedArg;
+  toBinding(value: T, options?: { createObj?: ObjCreator; updateMode?: UpdateMode }): binding.MixedArg;
```

Comment:
Is the `updateMode` being set somewhere?  I haven't yet seen this in any of the `toBinding` calls.

Reply:
Not yet, but that's the intention for the more ideal solution down the line. So I kept that refactor. (I'm mentioning it in [this TODO comment](https://github.com/realm/realm-js/blob/5e2caccff9c805205939bb4932e77d44eb6748f8/packages/realm/src/TypeHelpers.ts#L264).) Will also create a ticket and link to the ticket in that comment.

Reply:
Perfect 🙏🏼